client/ayon_houdini/plugins/load/show_usdview.py

```diff
@@ -22,7 +22,12 @@ def load(self, context, name=None, namespace=None, data=None):
         from pathlib import Path
 
         if platform.system() == "Windows":
-            executable = "usdview.bat"
+            houdiniVersion = os.environ["HOUDINI_VERSION"]
+            major = int(houdiniVersion.split(".")[0])
+            if major >= 20:
```

Comment:
I think we're better of querying this from Houdini directly.
```suggestion
            if hou.applicationVersion()[0] >= 20:
```

Do note that you'd need to add `import hou` too.